palindrome.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <string.h>
+
+int isPalindrome(char string[], int left, int right) {
+	if(left>=right) {
+		return 0;
+	}
+	else {
+		return 1;
+	}
+	if(string[left] == string[right]) {
```

Comment:
my guess is your error has something to do with the fact that this is a separate if statement